src/interfaces/UserInterface.ts

```diff
@@ -1 +1,9 @@
-export {}
+import mongoose from 'mongoose'
+
+export default interface UserInterface {
+  id: string
+  name: string
+  email: string
+  password: string
+  slides: [{ _id: mongoose.Types.ObjectId; title: string; link: string }]
```

Comment:
This should extracted into another interface called `Slide` into its own file `SlideInterface`